x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/TransactionBreakdownGraph/index.tsx

```diff
@@ -4,10 +4,62 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import React from 'react';
+import React, { useMemo, useCallback } from 'react';
+import numeral from '@elastic/numeral';
+import { NOT_AVAILABLE_LABEL } from '../../../../../common/i18n';
+import { Coordinate } from '../../../../../typings/timeseries';
+import { TransactionLineChart } from '../../charts/TransactionCharts/TransactionLineChart';
+import { asPercent } from '../../../../utils/formatters';
+import { unit } from '../../../../style/variables';
 
-const TransactionBreakdownGraph: React.FC<{}> = () => {
-  return <div />;
+interface Props {
+  timeseries: Array<{
+    name: string;
+    color: string;
+    values: Array<{ x: number; y: number | null }>;
```

Comment:
I think this should be consistent with the other charts we have (`name` -> `title` and `values` -> `data`).
As mentioned somewhere else I prefer `values` over `data` but might be a bigger job to change that everywhere.

Instead of having several different interfaces can we make a single timeseries interface that is re-used?